dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/OSUtils.java

```diff
@@ -118,11 +117,11 @@ public static double availablePhysicalMemorySize() {
      */
     public static double totalMemorySize() {
```

Comment:
Maybe it's better to change this method name to `totalPhysicalMemorySize`.

Reply:
Done